src/main/scala/com/lightbend/kafkalagexporter/KafkaClient.scala

```diff
@@ -150,12 +150,34 @@ class KafkaClient private[kafkalagexporter](cluster: KafkaCluster,
       groups <- adminClient.listConsumerGroups()
       groupIds = getGroupIds(groups)
       groupDescriptions <- adminClient.describeConsumerGroups(groupIds)
+      noMemberGroups = groupDescriptions.asScala.filter{ case (_, d) => d.members().isEmpty}.keys
+      noMemberGroupsPartitionsInfo <- Future.sequence(noMemberGroups.map(g => getGroupPartitionsInfo(g)))
     } yield {
       val gtps = groupDescriptions.asScala.flatMap { case (id, desc) => groupTopicPartitions(id, desc) }.toList
-      (groupIds, gtps)
+      val gtpsNoMembers = noMemberGroupsPartitionsInfo.flatten
+      (groupIds, gtps ++ gtpsNoMembers)
     }
   }
 
+  /**
+    * Retrieve partitions by consumer group ID. This is used in case when members info is unavailable for the group.
+    * @param groupId
+    * @return
```

Comment:
You can drop this

Reply:
@seglo Done dropping extra comment lines.
About the names for consumers: I was actually thinking about that, I can add them.
As for the names of the hosts, that might be somewhat misleading. Consumers are obviously distinct, and their names are made up anyway, so for them, auto-generated names are fine. Hosts, on the other hand, are real machines. I don't see any benefit of giving them fake names.
Please let me know what you think.

Reply:
Actually, on the second thought, it looks like even for the consumers generated names are not that useful. Giving them all distinct names may not reflect the real picture, because it could be one consumer for multiple partitions.
As for the test, unfortunately I would not have time for writing a formal test. However, I tested my solution manually in my environment, an the result is pretty obvious - I see it in the browser.

Reply:
I didn't mean the literal name for the consumers.  Just references in code.  For example, in the code and scaladocs, or a test name.

Reply:
Oh, I see. Up to you how to call it.
Honestly, when I was looking for the solution, I overlooked #124 because it was not obvious what 'simple consumer' means.
So currently I am playing with my own build here, but I would really prefer to  download an official distribution. Please let me know when do you think it can be ready and whether you want me to change anything in my code.
Thanks!